### DISCUSSION TITLE
Adding Buttons

### DISCUSSION THREAD

**Discussion by gkmundt** [michaelrsweet/htmldoc#518] [Q&A]
Is there any way to add a hyperlink as a button on the PDF.  I've tried

```html
<a href="https://www.mysite.htm">
  <img src="mybutton.jpg" alt="Click Here">
</a>
```

But it does not get treated as a button or a link.

**Comment by michaelrsweet** [comment]
What version of HTMLDOC are you running?

What options are you passing to htmldoc when creating your PDF?

**Comment by gkmundt** [comment]
Michael,

I am still on the older version and researching the ELF file used.  I have found a work around by creating a URL with the pdf files embedded.
